go/db/generate_patches.go

```diff
@@ -576,4 +576,9 @@ var generateSQLPatches = []string{
 			database_instance
 			ADD COLUMN semi_sync_available TINYINT UNSIGNED NOT NULL DEFAULT 0 AFTER semi_sync_enforced
 	`,
+	`
+		ALTER TABLE
```

Comment:
```suggestion
		ALTER TABLE /* sqlite3-skip */
```
required for `sqlite3` backend